fleetspeak/src/client/client_test.go

```diff
@@ -43,6 +52,7 @@ type fakeService struct {
 
 func (s *fakeService) Start(sc service.Context) error {
 	s.sc = sc
+	fakeServiceStartCount++
 	return nil
```

Comment:
Do we need to use an atomic int for the start/stop counters? These tests can be run concurrently.

Reply:
In this case not: the only scenario when global variables would be accessed by more than one thread is when multiple tests from this test suite will run in parallel. But running tests in parallel is likely to break the tests anyway: no matter if global counters are atomic are not.